### ISSUE TITLE
Repositories without slashes in path are ignored.

### ISSUE THREAD

**Issue by rczajka** [retr0h/git-url-parse#3]
Parsing any URL which doesn't contain a `/` in the path fails.

For example, this is correct (even if the 'owner' part isn't generally meaningful):

```
>>> giturlparse.parse('ssh://user@example.com/path/repo.git')
Parsed(pathname='/path/repo.git', protocols=['ssh'], protocol='ssh', href='ssh://user@example.com/path/repo.git', resource='example.com', user='user', port=None, name='repo', owner='path')
```

But this yields nonsensical results:

```
>>> giturlparse.parse('ssh://user@example.com/repo.git')
Parsed(pathname='/user@example.com/repo.git', protocols=['ssh'], protocol='ssh', href='ssh://user@example.com/repo.git', resource='ssh', user=None, port=None, name='repo', owner='user@example.com')
```

This, again, is correct:

```
>>> giturlparse.parse('user@example.com:path/repo.git')
Parsed(pathname='path/repo.git', protocols=[], protocol='ssh', href='user@example.com:path/repo.git', resource='example.com', user='user', port=None, name='repo', owner='path')
```

And this fails:

```
>>> giturlparse.parse('user@example.com:repo.git')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/rczajka/.local/share/virtualenvs/vdeploy/lib/python3.6/site-packages/git_url_parse-1.0.2-py3.6.egg/giturlparse/__init__.py", line 34, in parse
    return p.parse()
  File "/home/rczajka/.local/share/virtualenvs/vdeploy/lib/python3.6/site-packages/git_url_parse-1.0.2-py3.6.egg/giturlparse/parser.py", line 104, in parse
    raise ParserError(msg)
giturlparse.parser.ParserError: Invalid URL 'user@example.com:repo.git'
```

**Comment by rczajka** [comment]
Actually, all URLs [listed in tests as invalid](https://github.com/retr0h/git-url-parse/blob/master/test/conftest.py#L179) are valid git URLs:

        'https://example.com/owner',
        'git+ssh://example.com/owner/repo.git'
        'git+https://example.com/owner/repo.git'
        'https://example.com/owner',
        'example.com:repo.git',
        'user@example.com:repo.git',

Even the `git+https` one is valid, even though it'd require a remote helper for git to understand it. Other ones are totally ok.

**Comment by retr0h** [comment]
> Actually, all URLs listed in tests as invalid are valid git URLs:

They are invalid b/c I have chosen not to parse those as valid.  PRs always welcome.

**Comment by retr0h** [comment]
Implemented in #4.
